Rebranding docs/install/

zuiderkwast opened the pull request:
This PR does the bulk of renaming:

* Remove "redis" in CLI prompt "redis 127.0.0.1:6379>"
* Change "Redis" to "Valkey" using Perl regex `Redis(?! compatibility| v?[1-7]| version| < [1-7]| OSS|\.[a-z]|\w)`
* Replace filenames redis(-cli|-server|-benchmark|.conf) with valkey equivalents
* Replace /etc/redis and /var/redis with valkey equivalent paths
* "redis_6379" -> "valkey_6379"

Part of #18

stockholmux commented on docs/install/_index.md:
```diff
@@ -1,15 +1,15 @@
 Here are the installation instructions:
 
-* [Install Redis](/docs/install/install-redis)
+* [Install Valkey](/docs/install/install-redis)
```
That path doesn't look right.

stockholmux commented on docs/install/install-redis/install-redis-on-linux.md:
```diff
@@ -1,18 +1,18 @@
-Most major Linux distributions provide packages for Redis.
+Most major Linux distributions provide packages for Valkey.
 
 ## Install on Ubuntu/Debian
```
This is not correct as of today. We don't have a debian based package yet.

zuiderkwast replied:
Shall we remove it?

I was thinking we can also remove it after bulk-rebranding.

stockholmux replied:
Yeah, I somewhat expect that most of this content will change.

zuiderkwast replied:
Deleted.

stockholmux commented on docs/install/install-redis/install-redis-on-linux.md:
```diff
@@ -35,7 +35,7 @@ sudo apt-get install redis
 
 ## Install from Snapcraft
```
We don't have snapcraft yet. This section isn't accurate.

zuiderkwast replied:
I can remove this section.

stockholmux replied:
I think we should.

stockholmux commented on docs/install/install-redis/install-redis-on-windows.md:
```diff
@@ -1,23 +1,23 @@
 Microsoft provides [detailed instructions for installing WSL](https://docs.microsoft.com/en-us/windows/wsl/install). Follow these instructions, and take note of the default Linux distribution it installs. This guide assumes Ubuntu.
 
-## Install Redis
+## Install Valkey
```
No debian yet, so this section is not accurate.

zuiderkwast replied:
OK, I'll delete the section.

stockholmux commented on docs/install/install-redis/_index.md:
```diff
@@ -1,159 +1,159 @@
-* [Install Redis on Linux](/docs/install/install-redis/install-redis-on-linux)
-* [Install Redis on macOS](/docs/install/install-redis/install-redis-on-mac-os)
-* [Install Redis on Windows](/docs/install/install-redis/install-redis-on-windows)
+* [Install Valkey from Source](/docs/install/install-redis/install-redis-from-source)
```
These paths won't be right.

zuiderkwast replied:
They will be fixed in #30.

stockholmux reviewed (changes requested):
I think this was mostly just a search/replace PR but there are things that won't work here.

zuiderkwast commented on docs/install/_index.md:
```diff
@@ -1,15 +1,15 @@
 Here are the installation instructions:
 
-* [Install Redis](/docs/install/install-redis)
+* [Install Valkey](/docs/install/install-redis)
```
Well spotted, but one thing at a time.

I'll handle the filenames in another PR. I have a script that will rename all the files to their original names stored in aliases metadata, typically as /topics/install for this one. That script also checks that there are no broken links. I'll do that once the bulk rebranding is done and merged.

zuiderkwast replied:
#30

stockholmux replied:
Fair. Should we make it clear to reviewers what should be ignored or payed attention to?

zuiderkwast commented on docs/install/install-redis/_index.md:
AFAICT, the init script `utils/redis_init_script` hasn't been renamed or rebranded within the code repo. When it's done, we'll add the label `needs-doc-pr` to fix the docs to match the reality. That's the usual process anyway. I didn't want to do this in the reverse order since stuff may be forgotten and not be correct then.

zuiderkwast replied:
And I didn't want to do any style fixes of the old docs. Just rebranding.

stockholmux replied:
Fair on the style fix and I get the order on the init script. That was surprising it hasn't been done yet.

zuiderkwast replied:
There's more to do in the main repo. Some log messages still contain "Redis" I believe.

The problem with the init script template is that nobody knows if it really works. It's not tested in CI. :) But we should rebrand it anyway.

zuiderkwast replied:
https://github.com/valkey-io/valkey/issues/354

zuiderkwast commented:
I added info about docker containers under the linux install page.

I updated the mac page to say "brew install valkey" and other commands like "brew services start valkey". I hope it works. :)

Let's improve this information later. I just want to delete wrong info for now.

hwware commented on docs/install/install-redis/_index.md:
```diff
@@ -1,159 +1,159 @@
+This is a an installation guide. You'll learn how to install, run, and experiment with the Valkey server process.
 
-While you can install Redis on any of the platforms listed below, you might also consider using Redis Cloud by creating a [free account](https://redis.com/try-free?utm_source=redisio&utm_medium=referral&utm_campaign=2023-09-try_free&utm_content=cu-redis_cloud_users).
+While you can install Valkey on any of the platforms listed below, you might also consider using Valkey Cloud by creating a [free account](https://redis.com/try-free?utm_source=redisio&utm_medium=referral&utm_campaign=2023-09-try_free&utm_content=cu-redis_cloud_users).
```
We have no this kind of website so far, can we just remove this part?

hwware replied:
For this PR, I just add a minor comment here,

There are some links which include redis keyword, but i prefer updating them in a seperated pr as what @zuiderkwast said. Thanks
